daemon/daemon.go

```diff
@@ -368,8 +368,8 @@ func (daemon *Daemon) restore() error {
 				}
 			} else if !daemon.configStore.LiveRestoreEnabled {
 				logger(c).Debug("shutting down container considered alive by containerd")
-				if err := daemon.shutdownContainer(c); err != nil && !errdefs.IsNotFound(err) {
-					log.WithError(err).Error("error shutting down container")
+				if err := daemon.shutdownContainer(context.Background(), c); err != nil && !errdefs.IsNotFound(err) {
+					logrus.WithError(err).WithField("container", c.ID).Error("error shutting down container")
```

Comment:
why were these changed back to `logrus` ? Intentional?

Reply:
Brain didn't see that we were using a different log object.

Reply:
Fixed